pkgs/development/beam-modules/default.nix

```diff
@@ -43,27 +43,27 @@ let
         # BEAM-based languages.
         elixir = elixir_1_7;
 
-        elixir_1_8 = lib.callElixir ../interpreters/elixir/1.8.nix {
+        elixir_1_9 = lib.callElixir ../interpreters/elixir/1.9.nix {
           inherit rebar erlang;
           debugInfo = true;
         };
 
-        elixir_1_7 = lib.callElixir ../interpreters/elixir/1.7.nix {
+        elixir_1_8 = lib.callElixir ../interpreters/elixir/1.8.nix {
           inherit rebar erlang;
           debugInfo = true;
         };
 
-        elixir_1_6 = lib.callElixir ../interpreters/elixir/1.6.nix {
+        elixir_1_7 = lib.callElixir ../interpreters/elixir/1.7.nix {
           inherit rebar erlang;
           debugInfo = true;
         };
 
-        elixir_1_5 = lib.callElixir ../interpreters/elixir/1.5.nix {
+        elixir_1_6 = lib.callElixir ../interpreters/elixir/1.6.nix {
           inherit rebar erlang;
           debugInfo = true;
         };
 
-        elixir_1_4 = lib.callElixir ../interpreters/elixir/1.4.nix {
+        elixir_1_5 = lib.callElixir ../interpreters/elixir/1.5.nix {
```

Comment:
This diff looks strange, did you remove 1.4? If so also delete `1.4.nix`.

Reply:
I've deleted also 1.4.nix now.